packages/client-ts/src/client.ts

```diff
@@ -208,6 +210,39 @@ class Client implements IClient {
     return this.config
   }
 
+  public async getPublicProfile(did: string,
+    contextName: string,
+    profileName: string = "basicProfile",
+    fallbackContext: string | null = "Verida: Vault",
```

Comment:
Just to be sure.
It is intentional to allow `null` for the fallbackContext?
So that if set to `null`, we don't use the fallback, but if we let it `undefined`, we will fallback to the Vault, right?

Reply:
I known about this. Yes, this is the intention.

Reply:
Ok, thanks.
I prefer to make sure because using `null` doesn't apply the default value.